cmd/flowrunner/testdata/flows/all_actions.json

```diff
@@ -178,7 +178,7 @@
                         {
                             "uuid": "e0ae5679-5145-4bdd-9737-3481932bf095",
                             "type": "set_contact_language",
-                            "language": "eng"
+                            "language": ""
```

Comment:
needed to change language but have the flow still work.. so just clearing it instead which still generates a `contact_language_changed` event